Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InAppNotification/InAppNotificationCode.bind

```diff
@@ -0,0 +1,85 @@
+// ## XAML - Text + buttons (DataTemplate)
+
+<DataTemplate x:Key="InAppNotificationWithButtonsTemplate">
+    <UserControl>
+        <Grid DataContext="{Binding ElementName=CurrentPage}">
+            <VisualStateManager.VisualStateGroups>
+                <VisualStateGroup>
+                    <VisualState x:Name="NarrowState">
+                        <VisualState.Setters>
+                            <Setter Target="TextBlock.(Grid.RowSpan)" Value="1" />
+                            <Setter Target="TextBlock.(Grid.ColumnSpan)" Value="2" />
+
+                            <Setter Target="ButtonsStackPanel.(Grid.Row)" Value="1" />
+                            <Setter Target="ButtonsStackPanel.(Grid.RowSpan)" Value="1" />
+                            <Setter Target="ButtonsStackPanel.(Grid.Column)" Value="0" />
+                            <Setter Target="ButtonsStackPanel.(Grid.ColumnSpan)" Value="2" />
+                            <Setter Target="ButtonsStackPanel.Orientation" Value="Vertical" />
+                            <Setter Target="ButtonsStackPanel.Margin" Value="0 5 0 0" />
+
+                            <Setter Target="YesButton.Width" Value="Auto" />
+                            <Setter Target="YesButton.HorizontalAlignment" Value="Stretch" />
+
+                            <Setter Target="NoButton.Width" Value="Auto" />
+                            <Setter Target="NoButton.HorizontalAlignment" Value="Stretch" />
+                            <Setter Target="NoButton.Margin" Value="0 5 0 0" />
+                        </VisualState.Setters>
+                                    
+                        <VisualState.StateTriggers>
+                            <StateTrigger IsActive="{Binding IsRootGridActualWidthLargerThan700, Converter={StaticResource BoolNegationConverter}, Mode=OneWay}" />
+                        </VisualState.StateTriggers>
+                    </VisualState>
+
+                    <VisualState x:Name="LargeState">
+                        <VisualState.StateTriggers>
+                            <StateTrigger IsActive="{Binding IsRootGridActualWidthLargerThan700, Mode=OneWay}" />
+                        </VisualState.StateTriggers>
+                    </VisualState>
+                </VisualStateGroup>
+            </VisualStateManager.VisualStateGroups>
+
+            <Grid x:Name="RootGrid" SizeChanged="RootGrid_SizeChanged">
```

Comment:
To be clear, the `DataTemplate` available in the bind code file is a specific template that handle responsive design based on the container Width. And to make it work, I need a property that tell what StateTrigger I should use, in this case I used a boolean in the code behind.

Of course, it is not the default behavior and you can perfectly use the control without extra code : 

```
<controls:InAppNotification x:Name="ExampleInAppNotification" />
```
And then use `.ShowAsync()` method.

My objective with the `DataTemplate` was to provide a more complete example that mimic MS Edge notification with buttons (Horizontal Orientation when Width > 700px, Vertical Orientation otherwise).